qa/integration/services/logstash_service.rb

```diff
@@ -38,7 +38,10 @@ def initialize(settings)
       # First try without the snapshot if it's there
       @logstash_home = File.expand_path(File.join(LS_BUILD_DIR, ls_file), __FILE__)
       @logstash_home += "-SNAPSHOT" unless Dir.exists?(@logstash_home)
-
+      
+      @logstash_queue_path = File.join("#{@logstash_home}", "data")
```

Comment:
@suyograo damn busted on travis with:

```
rspec ./specs/01_logstash_bin_smoke_spec.rb:145 # Test Logstash instance should keep the same id between restarts

```
 will get on this in the morning again :)